infra-as-code/bicep/modules/subscriptionPlacement/subscriptionPlacement.bicep

```diff
@@ -1,12 +1,12 @@
 targetScope = 'managementGroup'
 
 @description('Array of Subscription Ids that should be moved to the new management group.')
-param parSubscriptionIds array = []
+param parSubscriptionIds array
```

Comment:
Possibility of breaking something ~ check

Reply:
Checked, and it is breaking because of the change from _optional_ to _required_.

Reply:
Could we change this back @johnlokerse so it's not changing. We use the technique quite often to limit parameter inputs required

Reply:
Resolved 👍